apps/studio/components/interfaces/ProjectCreation/RegionSelector.tsx

```diff
@@ -73,6 +83,10 @@ export const RegionSelector = ({
     process.env.NEXT_PUBLIC_ENVIRONMENT === 'local' ||
     process.env.NEXT_PUBLIC_ENVIRONMENT === 'staging'
 
+  if (isErrorAvailableRegions) {
```

Comment:
The selector can still work without the available regions, not sure if this is needed.